### ISSUE TITLE
bug: Dark mode is broken

### ISSUE THREAD

**Issue by kyoyacchi** [ReVanced/revanced-manager#1290]
### Bug description

I was using the app in dark mode until I updated the app to v1.10.1, but it no longer works. It consistently shows the light mode, whether I enable or disable dark mode or light mode. My phone model is Tecno Spark 7T (Android 11)

![Screenshot_20230921-112605](https://github.com/ReVanced/revanced-manager/assets/63583961/84baef59-68d7-4082-9169-6d62b6e755b0)
![Screenshot_20230921-112609](https://github.com/ReVanced/revanced-manager/assets/63583961/9d1515b9-f7d5-4e20-bdac-5278e59878ba)
![Screenshot_20230921-112611](https://github.com/ReVanced/revanced-manager/assets/63583961/ff1e32eb-6289-49c0-8e06-424b8a4156be)

Video(if needed):

https://github.com/ReVanced/revanced-manager/assets/63583961/2ab6f3fd-fa6e-4be2-9de3-fcc818df0c16

### Version of ReVanced Manager and version & name of application you tried to patch

ReVanced Manager v1.10.1
I didn't try any app to patch...

### Installation type

Non-root

### Device logs

```shell
--------- beginning of main
09-21 11:24:49.488 17108 17108 V PhoneWindow: DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@7f5eb08, this = DecorView@181b2a1[MainActivity]
09-21 11:24:49.508 17108 17108 V PhoneWindow: DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@7f5eb08, this = DecorView@181b2a1[MainActivity]
09-21 11:24:57.390 17108 17108 V PhoneWindow: DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@7f5eb08, this = DecorView@181b2a1[MainActivity]
09-21 11:24:57.429 17108 17185 I GED     : ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 2, oppidx_max 2, oppidx_min 0
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by BenjaminHalko** [comment]
This seems to be caused because for some reason manager has enabled Material You on a device that does not support it.

**Comment by luis-fss** [comment]
I'm also having this issue on Samsung Galaxy 9+

**Comment by validcube** [comment]
:tada: This issue have been fixed in #1293, and will be releasing soon:tm:.

:trollface: Sorry for permanently flashbanging everyone who was using this so called "dark" mode on Android 11 and under.
